packages/x-components/src/x-modules/empathize/components/empathize.vue

```diff
@@ -73,6 +76,23 @@
      */
     protected isOpen = false;
 
```

Comment:
I'll put it here, as you haven't added the new $ref to the $refs object.

```suggestion

    public $refs!: {
      /** Hidden span as default slot content to check if the empathize has content or not. */
      noContent?: HTMLSpanElement;
    };

```

Reply:
I don't see this being committed, did you forget about it? 😅

Reply:
🙈 Thks @LuisMartinez15 !